src/TLuaInterpreter.cpp

```diff
@@ -10082,7 +10082,12 @@ int TLuaInterpreter::setDiscordLargeIconText(lua_State* L)
         return warnArgumentValue(L, __func__, "access to Discord large icon text is disabled in settings for privacy");
     }
 
-    pMudlet->mDiscord.setLargeImageText(&host, getVerifiedString(L, __func__, 1, "text"));
+    auto discordText = getVerifiedString(L, __func__, 1, "text");
+    if (discordText.size() == 1) {
+        return warnArgumentValue(L, __func__, "text of length 1 not allowed");
```

Comment:
```suggestion
        return warnArgumentValue(L, __func__, "text of length 1 not allowed by Discord");
```

Just so people know that it's not Mudlet which has this random limitation

Reply:
Feedback applied 👍